cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1381,7 +1398,11 @@ public void run() {
         if (_executionException != null) {
           LOG.info("User task {}: Execution failed: {}. Exception: {}", _uuid, executionStatusString, _executionException.getMessage());
         } else {
-          LOG.info("User task {}: Execution succeeded: {}. ", _uuid, executionStatusString);
+          String status = "succeeded";
+          if (userTaskInfo != null && userTaskInfo.state() != COMPLETED) {
+            status = userTaskInfo.state().toString();
```

Comment:
In which scenario that it is not treat as succeeded when executionException==null?

Reply:
This happens if the user requests to stop the execution.

Reply:
Got it. Let's add a comment there

Reply:
Comment added.